lib/matplotlib/backend_bases.py

```diff
@@ -1743,7 +1743,7 @@ def _fix_ipython_backend2gui(cls):
         # `ipython --auto`).  This cannot be done at import time due to
         # ordering issues, so we do it when creating a canvas, and should only
         # be done once per class (hence the `lru_cache(1)`).
-        if "IPython" not in sys.modules:
+        if sys.modules.get("IPython", None) is None:
```

Comment:
You don't need the `, None)` as `get` already defaults to returning None for absent keys.

Approving modulo that change.

Reply:
I made the change you suggested: d70d826fa06bd44373d8687ca09bcee32fa8d155